src/server/data/index-data.json

```diff
@@ -1,51 +1 @@
-{
-  "biography":"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum",
-  "researchInterests":[
-   "A Really good text A Really good text A Really good text from data 111",
-   "A Really good text A Really good text A Really good text from data 222",
-   "A Really good text A Really good text A Really good text from data 333"
-  ],
-  "featuredPublications":[
-     {
-        "name":"Paper's Paper's title",
-        "description":"Paper's description Paper's description",
-        "fileLinks":[{"info": "download", "type": 1, "link": "aaa"}, 
-           {"info": "cite", "type": 3, "link": "bbb"}],
-    "authors": "first name, last name; first name, last name",
-        "year": 2019,
-        "topic":"cool topic",
-        "type":"paper",
-        "publisher":"good publisher good publisher",
-        "publicationStatus":"Published"
-     },
-     {
-        "name":"Paper's title Paper's title Paper's title Paper's title",
-        "description":"Paper's description Paper's description Paper's description Paper's description Paper's description Paper's description Paper's description Paper's description Paper's description Paper's description Paper's description",
-        "fileLinks":[{"info": "download", "type": 1, "link": "aaa"}, 
-           {"info": "cite", "type": 3, "link": "bbb"}],
-    "authors": "first name, last name; first name, last name",
-        "year": 2020,
-        "topic":"cool topic",
-        "type":"paper",
-        "publisher":"good publisher good publisher",
-        "publicationStatus":"Published"
-     }
-  ],
-  "currentProjects":[
-     {
-        "name":"Name 1",
-        "description":"Short Description Short Description Short Description ",
-        "link":{"info": "view project", "type": 1, "link": "aaa"}
-     },
-     {
-        "name":"Name 2",
-        "description":"Short Description Short Description Short Description ",
-        "link":{"info": "view project", "type": 1, "link": "aaa"}
-     },
-     {
-        "name":"Name 3",
-        "description":"Short Description Short Description Short Description ",
-        "link":{"info": "view project", "type": 1, "link": "aaa"}
-     }
-  ]
-}
\ No newline at end of file
+{"biography":"44","name":"444","age":"44","featuredPublications":[{"name":"65","description":"6","link":"66","type":"6"},{"name":"61","description":"2","link":"13","type":"21"}],"currentProjects":[{"name":"ok","description":"ok","link":"ok"}]}
```

Comment:
I am not sure this is the right format...

Reply:
that was just a demo. i'll adjust the fields